Reskin: Fix #2008 - Dropdowns on create entity not displaying properly

gkrishnan724 opened the pull request:
@mbj36  
Thanks for pointing out that mistake I did not check properly. Here is the proper PR

mbj36 commented:
@gkrishnan724 There is still a problem there. Replicate here at https://demo.openmf.org/#/entitydatatablechecks and see the difference

@santoshmath Any inputs ?

gkrishnan724 commented:
@mbj36  
I have fixed it, can you check it now?. 
Here are the test cases:

- Product field should only open when entity is either loan or savings account 

- Dropdowns populate according to the chosen entity

Also fixed the plus and trash icons.

Here's the screenshot:
![screenshot from 2017-03-10 12-48-17](https://cloud.githubusercontent.com/assets/21004514/23785962/43f85612-0590-11e7-88d7-f027d5512d17.png)
